backend/core/types/src/backend-swagger.ts

```diff
@@ -3722,6 +3722,9 @@ export interface UserCreate {
 
   /**  */
   phoneNumber?: string
+
+  /**  */
+  emailSubscription: string
```

Comment:
Should there be a corresponding change in user-create.dto.ts?

Reply:
I don't see references to all of the other variables in UserCreate in user-create.dto.ts, so I didn't think there needed to be a corresponding change but not totally sure

Reply:
The other variables are derived from User entity, which defines the user_accounts database. This file should really only be modified when running `yarn generate:client`.

Maybe we could leave this out until we figure out how we want to add these preferences into postgres?

Reply:
Oh I see, it's needed from the name of the field. In that case, you can just add the field to UserCreateDto, and then update the swagger client: https://github.com/CityOfDetroit/bloom/wiki/Regenerate-backend-swagger.ts

Reply:
ok added!